### DISCUSSION TITLE
Does QXlsx  support *.xls format excel ?

### DISCUSSION THREAD

**Discussion by QingJingYouYuan** [QtExcel/QXlsx#364] [Q&A]
I have a Excel 97-2003 format (.xls)  file need to be read and write.
My question is 
1. Does QXlsx support  97-2003 format (.xls)  file ?
2. Is there any different code process between   (.xls)  and (.xlsx)  ?
Thanks a lot!

**Comment by j2doll** [comment]
Dear @QingJingYouYuan 

The xls format is not supported. 
The xls format is a file in binary format. xlsx is a zip compressed file based on xml that is used as an iso standard.
